### ISSUE TITLE
Gamepad API support

### ISSUE THREAD

**Issue by Timmmm** [tagpro-game/tagpro-issues#287]
Tagpro should support the gamepad API.

I'm making a BLE joystick using a PSP-style slider specifically to play TagPro on my phone. Sending the joystick position would be much nicer than emulating key presses.

If someone can point me to the place in the code where thrust is applied to the ball I can implement it.

**Comment by I-mess-up** [comment]
I've played TagPro with a game pad before (Steam Controller, using the analog stick to emulate WASD). It was a really bad experience because the joystick doesn't give you as much control as an actual keyboard. Even if the devs were planning to implement game pad support, I would not recommend playing the game that way.

**Comment by Timmmm** [comment]
Yes but that's presumably because keyboard emulsion only allows 8 directions. That's exactly why I want true gamepad support.

**Comment by Nick-Riggs** [comment]
TagPro doesn't support partial acceleration values. So it isn't going to work well with an analog stick.

**Comment by Timmmm** [comment]
Aw that's a shame. Guess I'll have the joystick emulate it by tapping the keys on and off then (basically PWM).
